site/app/templates/grading/electronic/Details.twig

```diff
@@ -94,7 +103,7 @@
     <button type="button" class="btn btn-primary" onclick="expandAllSections()">Expand All Sections</button>
     {% if message is not empty %}
     <div class="details-warning-box{% if message_warning %} details-warning-highlight{% else %} details-warning-neutral{% endif %}">
-        {{ message }}
+        {{ message|nl2br }}
```

Comment:
maybe add the br in an {%if %} {%else%} condition, because I don't see this syntax a lot in the code, but I don't know it's probably ok